android/filament-android/src/main/java/com/google/android/filament/android/UiHelper.java

```diff
@@ -173,12 +174,36 @@ public void setDesiredSize(int width, int height) {
         }
     }
 
-    public int getDesiredWidth() {
-        return mDesiredWidth;
+    public void setDesiredWidth(int desiredWidth) {
```

Comment:
I'm confused, there shouldn't be a setDesiredWidth/Height(), that's done in setDesiredSize(), which calls mRenderSurface.resize() -- who calls mRenderSurface.resize() now?
Also why did getDesiredWidth()/Height() disappear?

Reply:
I messed up a refactoring and apparently did not amend the change after fixing it. The getters are supposed to still be here and there shouldn't be setDesiredWidth/Height.

Reply:
Fixed